package.json

```diff
@@ -21,14 +22,13 @@
     "doc": "doc",
     "test": "tests"
   },
-  "repository": "https://github.com/emberjs/ember-test-helpers",
   "scripts": {
     "build": "ember build",
     "changelog": "lerna-changelog",
     "docs": "scripts/precompile.sh && documentation build --document-exported \"addon-test-support/@ember/test-helpers/index.js\" --config documentation.yml -f md -o API.md && ember ts:clean",
-    "lint": "eslint *.js addon addon-test-support app config lib node-tests server tests test-support",
+    "lint": "eslint .",
     "start": "ember serve",
-    "test": "ember test",
+    "test": "EMBER_CLI_IGNORE_ADDON_NAME_MISMATCH=1 ember test",
```

Comment:
is this the right fix? 🤔

Reply:
Ya, lets go ahead and change `index.js` to do the blueprint thing:

```json
  name: require('./package').name,
```